capstone-ui/src/Components/ResultsBox/ResultsBox.jsx

```diff
@@ -60,15 +61,17 @@ export default function ResultsBox( { locationLevel, countyFIPS, selectedCandida
     candidates.sort((a, b) => b.voteCount - a.voteCount);
 
     function handleClick(key, name) {
-      setSelectedCandidates((selected) => {
-          const foundCandidate = selected.findIndex((candidate) => candidate.key === key);
-          if (foundCandidate !== -1) {
-              return selected.filter((candidate) => candidate.key !== key);
-          } else {
-              const updatedCandidates = selected.concat({ key, name });
-              return updatedCandidates.slice(-2);
-          }
-      });
+      if (!unclickables.includes(name)) {
```

Comment:
This works in terms of what your intention is here. However, I'm not sure how useful it is rendering these types of candidates. I don't know enough in this area of politics to understand why there are these 3 types. "Independent" & "Write-ins" seem ok to see in results, but "None of these candidates" seems off putting in result list of candidates.

Up to you if you want to leave as is. I'll approve as this will prevent these types to be added to followed candidates for a user where you will have issues later.

Reply:
I could maybe re-word "None of these candidates" as one of the other two options in the results boxes.